content/webapp/pages/article-series.tsx

```diff
@@ -117,6 +119,10 @@ export const getServerSideProps: GetServerSideProps<Props | AppErrorProps> =
         gaDimensions: {
           partOf: series.seasons.map(season => season.id),
         },
+        pageview: {
+          name: 'stories',
```

Comment:
@harrisonpim should this be singular?

Reply:
yeah, if we're following the existing pattern for works and images then we should use `stories` for the search results, and `story` for the thing in the list of results which gets clicked on - i think that makes this (and some of the others) a `story`